src/PointwiseFunctions/GeneralRelativity/ComputeSpacetimeQuantities.hpp

```diff
@@ -157,3 +157,44 @@ template <size_t SpatialDim, typename Frame, typename DataType>
 tnsr::A<DataType, SpatialDim, Frame> compute_spacetime_normal_vector(
     const Scalar<DataType>& lapse,
     const tnsr::I<DataType, SpatialDim, Frame>& shift) noexcept;
+
+/*!
+ * \ingroup GeneralRelativityGroup
+ * \brief  Computes extrinsic curvature from metric and derivatives.
+ * \details Uses ADM evolution equation for spatial metric.
+ */
+template <size_t SpatialDim, typename Frame, typename DataType>
+tnsr::ii<DataType, SpatialDim, Frame> compute_extrinsic_curvature(
+    const Scalar<DataType>& lapse,
+    const tnsr::I<DataType, SpatialDim, Frame>& shift,
+    const tnsr::iJ<DataType, SpatialDim, Frame>& deriv_shift,
+    const tnsr::ii<DataType, SpatialDim, Frame>& spatial_metric,
+    const tnsr::ii<DataType, SpatialDim, Frame>& dt_spatial_metric,
+    const tnsr::ijj<DataType, SpatialDim, Frame>&
+        deriv_spatial_metric) noexcept;
+
+/*!
+ * \ingroup GeneralRelativityGroup
+ * \brief  Computes generalized harmonic gauge source function.
+ * \details If \f$N, N^i, g_{ij}, \Gamma_{ijk}, K\f$ are the lapse, shift,
+ * spatial metric, spatial Christoffel symbols, and trace of the extrinsic
+ * curvature, then we compute
+ * \f{align}
+ * H_l &=& N^{-2} g_{il}(\partial_t N^i - N^k \partial_k N^i)
```

Comment:
I think you want just a single `&` after the `=`. Have you rendered this? I would expect the current case to have awkward spacing around the equal signs